src/mono/wasm/runtime/types/emscripten.ts

```diff
@@ -62,4 +66,4 @@ export declare interface EmscriptenModule {
     instantiateWasm: (imports: any, successCallback: Function) => any;
 }
 
-export declare type TypedArray = Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array | Uint32Array | Float32Array | Float64Array;
\ No newline at end of file
+export declare type TypedArray = Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | Uint16Array | Int32Array | Uint32Array | Float32Array | Float64Array;
```

Comment:
You can also use ArrayBufferView in place of this, though it has the downside of that interface inexplicably not having .length. Just an FYI, I'm fine with us defining a TypedArray alias (do the [bigint arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt64Array) need to go in here too?)